web/documentserver-example/ruby/app/models/file_model.rb

```diff
@@ -350,6 +351,31 @@ def get_users_mentions
     return !@user.id.eql?("uid-0") ? Users.get_users_for_mentions(@user.id) : nil
   end
 
+  def get_users_info
+    users_info = []
+    if !@user.id.eql?("uid-0")
+      Users.get_all_users().each do |user_info|
+        u = {
+          id: user_info.id,
+          name: user_info.name,
+          email: user_info.email,
+          group: user_info.group, 
+          reviewGroups: user_info.reviewGroups, 
+          commentGroups: user_info.commentGroups, 
+          userInfoGroups: user_info.userInfoGroups, 
+          favorite: user_info.favorite,
+          deniedPermissions: user_info.deniedPermissions, 
+          descriptions: user_info.descriptions, 
+          templates: user_info.templates, 
+          avatar: user_info.avatar
+        }
+        u["image"] = user_info.avatar ? "#{DocumentHelper.get_server_url(true)}/assets/#{user_info.id}.png" : nil
```

Comment:
Same. Looks like you missed the `images` subdirectory in the path.

Reply:
> Same. Looks like you missed the `images` subdirectory in the path.
Same